From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Removed remote site reference for logo.

Although the documentation demonstrates remote logo referencing, this is
regarded as a privacy violation in Debian. To ensure that the example is
still illustrative, a local version of the logo is used instead in the
actual markup that is converted to HTML.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 docs/user/markdown.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/user/markdown.rst b/docs/user/markdown.rst
index a43a676..dab51a4 100644
--- a/docs/user/markdown.rst
+++ b/docs/user/markdown.rst
@@ -424,7 +424,7 @@ do not appear in the rendered HTML:
 
 To transclude image from remote site:
 
-.. image:: http://static.moinmo.in/logos/moinmoin.png
+.. image:: moinmoin.png
    :alt: remote image
    :align: right
 
